bedrock/settings/base.py

```diff
@@ -631,8 +650,24 @@ def get_app_name(hostname):
     "django_jinja_markdown",
     "django_jinja",
     "watchman",
+    # Wagtail CMS and related, necessary apps'wagtail.contrib.forms',
```

Comment:
@stevejalim I was looking through this PR gathering things for us to use for our equivalent task in SuMO -- thank you for leading the way! 🙌 -- and just happened to notice what appears to be a typo that pulled `'wagtail.contrib.forms',` into the comment.

```suggestion
    # Wagtail CMS and related, necessary apps
    "wagtail.contrib.forms",
```

Reply:
Thanks, @escattone !!